### DISCUSSION TITLE
万言万当，不如一默 | /cn/2020/07/silence/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1377] [Comments]
https://yihui.org/cn/2020/07/silence/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
总结得很好。

统计一下每篇文章的下划线频率，按时间预测一下 3 什么时候会实现（应该是有上涨趋势的）。按我的联想力，实现这个也未必不可能，不过暂时还不想挑战这种全盘蜘蛛网式的写法，尽管现在蜘蛛网已经拉得有点密了。

> ——原帖发布于 2020-07-23 02:38:56

**Reply by yihui** [reply]
对，第一遍读的时候千万不要点开链接，否则就跟看维基百科页面一样，看着看着就忘了之前在看什么了。通常来说，我的链接只是起备注的作用，不看也罢，不会损失什么。

> ——原帖发布于 2021-01-19 14:54:44

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
提问的时候请提供一下可重复的例子，光是描述症状很难让人给出确切答案或者告诉你具体怎么解决。根据这点信息，我只能猜你有个 layouts/blogdown/postref.html 文件没有加到你的 Github 库中。

> ——原帖发布于 2021-01-09 15:48:05

**Reply by yihui** [reply]
问题解决了就好。

> ——原帖发布于 2021-01-10 03:17:46

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
居然来了个校友。我以前也一样觉得作业好多好难，还好 <https://web.archive.org> 能找到一部分老师往年的答案存档，还能抄抄续命。

> ——原帖发布于 2021-03-26 02:12:57

[a comment by an automated account is not shown]

**Comment by hilaily** [comment]
哈哈，我前段时间正在纠结要不要开个博客来记录自己的所思所想，但是写了东西又担心会社死。最终还是决定开一个，理由也是[求诸己](https://write.laily.net/post/2023-09-04-why-i-want-to-write/)，我需要一个地方来沉淀、记录我自己（到不一定要人看到，得到喝彩），今天正好发现了你的博客，感觉这就是我想要的样子。又发现你也是宜昌人，更惊喜了。

**Reply by yihui** [reply]
啊，居然来了个老乡，握爪！

说到社死，我恰好觉得写博客是对抗它的手段之一。社死之所以能发生，是因为无知的大众对那个人的了解太少，当他们抓住了一个可以批判的点之后，以为那就是那个人的全部，而实际上每个人都是极其多面的，是不能用某个时刻的某个表现去评判的。也许你会某篇博客中某个段落里某句话会表达出一个偏激的观点，但总体而言我相信你不会篇篇”踩雷“、句句”犯错“吧。

又及：”[我与我周旋久](https://write.laily.net)“这句话我是两个月前看[黄永玉的一篇纪念文章](https://www.thepaper.cn/newsDetail_forward_23497116)才第一次看到的，很有意思。祝你可以坚持写下去。

**Comment by ZaiyanHere** [comment]
我最近以「快节奏」的方式做了一些对自己很不友好的事情，累到情绪上要崩溃了才意识到这样的方式对自己多么不友好。一边慢慢读这一篇一边嗷嗷大哭。很感激这次邂逅。

**Comment by Leonewu** [comment]
只求心静

**Comment by shuibaco** [comment]
一大清早给我看清醒了……

我现在也在试图调整生活节奏，因为速度太快的话会不堪重负。但现在有个两难局面：投入生活的话，可以记录的事情变多，但没时间写博客；专心写博客的话会占用不少时间，体验生活的时间又少了。

我现在还是选择多和家人一起体验生活，所以博客素材一大推，但是完全没空更。没空也就没空了，以后再说哈哈哈😅

**Reply by yihui** [reply]
嘿，你还别说，一大晚上给我自己都看清醒了……
